app/classifier/tasks/selection/index.cjsx

```diff
@@ -0,0 +1,84 @@
+React = require 'react'
+GenericTask = require '../generic'
+SelectionTaskEditor = require './editor'
+
+Summary = React.createClass
+  displayName: 'SelectionSummary'
+
+  getDefaultProps: ->
+    task: {}
+    annotation: {}
+
+  render: ->
+    {answers} = @props.task
+
+    <div className="classification-task-summary">
+      <div className="question">{@props.task.instruction}</div>
+
+      <div className="answers">
+        {if @props.annotation.value
+          Object.keys(answers).map (key, i) =>
+            <div key={i} className="answer">
+              <i className="fa fa-arrow-circle-o-right" /> {@props.annotation?.value[key]}
+            </div>
+        }
+      </div>
+    </div>
+
+module?.exports = React.createClass
+  displayName: 'SelectionTask'
+
+  statics:
+    Editor: SelectionTaskEditor
+    Summary: Summary
+
+    getDefaultTask: ->
+      type: 'selection'
+      instruction: 'Select options from a list'
+      help: 'Click on the select box to choose an option'
+      answers: {}
+      answersOrder: []
+
+    getTaskText: (task) ->
+      task.instruction
+
+    getDefaultAnnotation: ->
+      value: {}
+
+    isAnnotationComplete: (task, annotation) ->
+      answer = (key) -> annotation.value[key]
+      answersCompleted = Object.keys(annotation.value)
+        .map(answer)
+        .filter(Boolean)
+
+      answersCompleted.length and (answersCompleted.length is Object.keys(task.answers).length)
+
+  render: ->
+    {answers, answersOrder} = @props.task
+    selectBoxes = if answersOrder.length then answersOrder else Object.keys(answers)
+
+    <GenericTask question={@props.task.instruction} help={@props.task.help} required={@props.task.required}>
+      <div className="selection-task">
+
+        {selectBoxes.map (name, i) =>
+          <select key={i} defaultValue="" ref="select-#{name}" onChange={@onChangeSelect.bind(@, selectBoxes)}>
```

Comment:
Ah good call I was only testing with a single task. Added in 55a66f659126b5b6a76ce58c70ff73a93e4ac186 and also testable in `/projects/alex-panoptes/bobbys-from-nature/classify`